specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/securitydomain.json

```diff
@@ -0,0 +1,244 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "KeyVaultClient",
+        "description": "The key vault client performs cryptographic key operations and vault operations against the Key Vault service.",
+        "version": "7.2-preview"
+    },
+    "x-ms-parameterized-host": {
+        "hostTemplate": "{vaultBaseUrl}",
+        "useSchemePrefix": false,
+        "positionInOperation": "first",
+        "parameters": [
+            {
+                "name": "vaultBaseUrl",
+                "description": "The vault name, for example https://myvault.vault.azure.net.",
+                "required": true,
+                "type": "string",
+                "in": "path",
+                "x-ms-skip-url-encoding": true
+            }
+        ]
+    },
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "paths": {
+        "/securitydomain/download": {
+            "post": {
+                "tags": [
+                    "HSMSecurityDomain"
+                ],
+                "operationId": "HSMSecurityDomainDownload",
+                "description": "Retrieves Security domain from HSM enclave",
+                "parameters": [
+                    {
+                        "in": "body",
+                        "name": "Certificates",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/CertificateSet"
+                        },
+                        "description": "Security domain download operation requires customer to provide three certificates in PEM format"
```

Comment:
I think we need to provide more details here: These certificates contain public key of RSA 2048/3072/4096-bit key-pair customer generated on-premise.

Reply:
Do we need a type classifier for the certificates? For example, is it okay to pass a base64 encoded PFX formatted certificate? Or can we make PEM format mandatory. @amitbapat, what should the user experience be?

Reply:
check out customer experience of the command 'az keyvault key download' which is reverse of what we are doing here. The CLI sends a 'Key Get", and service returns a JWK with public key. CLI converts JWK and writes out a PEM file. In the above case the experience is reverse order: Customer provides public key in PEM file, CLI converts it to JWK and send it.

Reply:
I really don't see any benefit to increasing complexity of the client code to take a PEM-encoded cert, encapsulate it in a JWK, unpack it at the other end as a PEM-encoded cert, and then just use it. If the JWK actually helped us do something better, we could. I also don't see any benefit to having it be a JWK in transit over a PEM file. I'm not an expert on JWK, so if there's something I'm missing, let me know. Just seems to be adding more code for no benefit.